### PR TITLE
Mg write procedures doc

### PR DESCRIPTION
**Pull request by antaljanosbenjamin**

### CODE REVIEW

File: docs/database-functionalities/query-modules/implement-query-modules.md [hunk 1]
```diff
@@ -145,6 +145,44 @@ return mgp.Record(args=args_copy, vertex_count=vertex_count,
                   avg_degree=avg_degree, props=props)
 ```
 
+### Writeable procedure
+
+Writeable procedures can be implemented in very similar way as read-only
```
Comment (jbajic) [inline docs/database-functionalities/query-modules/implement-query-modules.md:150]:
```suggestion
Writeable procedures can be implemented in a very similar way as read-only
```

File: docs/database-functionalities/query-modules/implement-query-modules.md [hunk 2]
```diff
@@ -145,6 +145,44 @@ return mgp.Record(args=args_copy, vertex_count=vertex_count,
...
+procedures. The only difference is writeable procedures receive mutable objects,
+therefore they can create or delete vertices or edges, modify the properties of
+vertices and edges and they can add or remove labels of vertices.
+
+We can implement a very simple writeable query modules similarly to read-only
+procedures:
```
Comment (jbajic) [inline docs/database-functionalities/query-modules/implement-query-modules.md:156]:
```suggestion
We can implement a very simple writeable query module similarly to read-only
procedures:
```

File: docs/database-functionalities/query-modules/implement-query-modules.md (hunk 1 above)
Comment (kostasrim) [inline docs/database-functionalities/query-modules/implement-query-modules.md:150]:
```suggestion
Writeable procedures can be implemented in a very similar way as read-only
```

File: docs/database-functionalities/query-modules/implement-query-modules.md [hunk 3]
```diff
@@ -145,6 +145,44 @@ return mgp.Record(args=args_copy, vertex_count=vertex_count,
                   avg_degree=avg_degree, props=props)
 ```
 
+### Writeable procedure
+
+Writeable procedures can be implemented in very similar way as read-only
+procedures. The only difference is writeable procedures receive mutable objects,
+therefore they can create or delete vertices or edges, modify the properties of
```
Comment (kostasrim) [inline docs/database-functionalities/query-modules/implement-query-modules.md:152]:
```suggestion
therefore they can create and delete vertices or edges, modify the properties of
```

File: docs/database-functionalities/query-modules/implement-query-modules.md [hunk 4]
```diff
@@ -145,6 +145,44 @@ return mgp.Record(args=args_copy, vertex_count=vertex_count,
...
+vertices and edges and they can add or remove labels of vertices.
+
+We can implement a very simple writeable query modules similarly to read-only
```
Comment (kostasrim) [inline docs/database-functionalities/query-modules/implement-query-modules.md:155]:
```suggestion
We can implement a very simple writeable query module similarly to read-only
```

File: docs/database-functionalities/query-modules/implement-query-modules.md [hunk 5]
```diff
@@ -145,6 +145,44 @@ return mgp.Record(args=args_copy, vertex_count=vertex_count,
...
+    # Connect the new vertex to the other vertices
+    for v in vertices_to_connect:
+        context.graph.create_edge(vertex, v, mgp.EdgeType("HAS_SAME_VALUE"))
+
+    return mgp.Record(created_vertex=vertex)
+
+```
+
+This example procedure creates a new vertex with the specified property and
+connects it to all existing vertex which has the same property with the same
```
Comment (kostasrim) [inline docs/database-functionalities/query-modules/implement-query-modules.md:182]:
```suggestion
connects it to all existing vertices which have the same property with the same
```

**Comment by gitbuda** [comment]
@antaljanosbenjamin is the CHANGELOG update missing or it's somewhere else?

**Comment by antaljanosbenjamin** [comment]
@gitbuda I forgot to push yesterday, did it now.

File: docs/database-functionalities/query-modules/implement-query-modules.md (hunk 1 above)
Comment (antaljanosbenjamin) [reply docs/database-functionalities/query-modules/implement-query-modules.md]: @jbajic had the same comment, already applied his changes.

File: docs/database-functionalities/query-modules/implement-query-modules.md (hunk 5 above)
Comment (antaljanosbenjamin) [reply docs/database-functionalities/query-modules/implement-query-modules.md]: @jbajic had the same comment, already applied his changes.

**Review by g-despot — APPROVED** [review]

**Pull request merged**